warehouse/bigquery/bigquery.go

```diff
@@ -396,17 +396,31 @@ func (bq *HandleT) loadTable(tableName string, _, getLoadFileLocFromTableUploads
 		primaryJoinClause := strings.Join(primaryKeyList, " AND ")
 		bqTable := func(name string) string { return fmt.Sprintf("`%s`.`%s`", bq.namespace, name) }
 
+		var orderByClause string
+		if _, ok := tableColMap["received_at"]; ok {
+			orderByClause = "ORDER BY received_at DESC"
```

Comment:
@achettyiitr why it was a problem to always do `ORDER BY received_at DESC`?

Reply:
When enabling ID resolution, we create 2 tables.
1. rudder_identity_mappings (merge_property_type, merge_property_value)
2. rudder_identity_merge_rules (merge_property_1_type, merge_property_1_value, merge_property_2_type, merge_property_2_value)